### PR TITLE
Improve docs tabs styling

### PR DESCRIPTION
**Pull request by laurakalbag**
Improves the docs tabs styling to make it more readable and compact, as well as working better with code examples.

**Preview at: https://docs-kb12h8onu-statelyai.vercel.app/xstate-v5/migration#breaking-changes**

Yes, I really wish I could make it borders-only and not use borders on the bottom of the active tab… but I can’t find a flexbox-compatible way to do so 🥲 

Before:
CleanShot 2023-05-11 at 17 26 55@2x
CleanShot 2023-05-11 at 17 26 49@2x

After:
CleanShot 2023-05-12 at 16 55 30@2x
CleanShot 2023-05-12 at 16 55 22@2x

### CODE REVIEW

File: versioned_docs/version-5/url.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: Share machines using their share URL
+---
+
+You can share machines with your team or publicly with the machine URL.
+
+Your machine will only be viewable if:
+
+- the project visibility is **public**
+- the project visibility is **unlisted**
+- the project visibility is **private**, and the viewer is also a member of your team
```
Comment (kevinmaes) [inline versioned_docs/version-5/url.mdx:11]: I now see this example with 3 bullet points which makes perfect sense. Maybe we should list bullet points for 2 or more items (in reference to my other comment).

**Review by kevinmaes — APPROVED** [review]
These look great. The tabs look much nicer now!

**Comment by kevinmaes** [comment]
@laurakalbag I pushed 2 commits, one for removing the horizontal dividing lines and increasing the tab bottom margin and the other to fix typos with `:::tip` admonitions.

File: versioned_docs/version-5/url.mdx (hunk 1 above)
Comment (laurakalbag) [reply versioned_docs/version-5/url.mdx:11]: Yeah, I’m not sure how one bullet point happened. I like bullets for breaking information into a more skimmable format, it also serves to mix up the formatting a bit so it’s not endless paragraphs!

**Pull request merged**